templates/vue/src/components/tyde/activities/CircleOfSupport/AddCommunityForm.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <div class="content">
+    <b-overlay :show="isLoading" style="width: 100%">
+      <form @submit.stop.prevent>
+        <div class="community" style="flex: 1">
+          <input
+            v-model="community.name"
+            name="community name"
+            class="community-name"
+            type="text"
+            placeholder="community"
+            @blur="isInputTouched = true"
+          />
+          <b-form-invalid-feedback :state="isNameValid">
+            Please enter a name for your community.
+          </b-form-invalid-feedback>
+          <div id="emoji-picker" style="position: relative">
+            <button class="preview" @click="showPicker = !showPicker">
+              {{ community.icon }}
+            </button>
+            <div v-show="showPicker" class="picker" data-qa="emoji-picker">
+              <v-emoji-picker @select="community.icon = $event.data" />
+            </div>
+          </div>
+          <div class="controls">
+            <button @click="$emit('back')">Cancel</button>
+            <button class="submit" @click="addCommunity">
+              Add community
+            </button>
+          </div>
+        </div>
+        <div
+          class="community-preview"
+          :style="{ flex: 1, '--color': community.color }"
+        ></div>
+        <ul class="colors" style="flex: 1">
+          <li v-for="color in colors" :key="color" :style="{ '--color': color }">
+            <button
+              :class="['color', { active: color === community.color }]"
+              @click="community.color = color"
+            ></button>
+          </li>
+        </ul>
+      </form>
+    </b-overlay>
+  </div>
+</template>
+
+<script>
+import { VEmojiPicker } from "v-emoji-picker"
+import client from "@/services/TapestryAPI"
+
+export default {
+  components: {
+    VEmojiPicker,
+  },
+  data() {
+    return {
+      community: {
+        name: "",
+        icon: "👨‍👩‍👦",
+        color: "",
```

Comment:
Should a default color be set? Otherwise, color is stored as "" in the db.

Reply:
@wynnset thoughts?